packages/core/admin/ee/server/utils/review-workflows.js

```diff
@@ -3,17 +3,23 @@
 const { WORKFLOW_MODEL_UID } = require('../constants/workflows');
 
 /**
- * Determine if a content type has the review workflows feature enabled
- * @param {Object} contentType
- * @returns
+ * Checks if a content type has review workflows enabled.
+ * @param {string|Object} contentType - Either the modelUID of the content type, or the content type object.
+ * @returns {boolean} Whether review workflows are enabled for the specified content type.
  */
-const hasRWEnabled = (contentType) => contentType?.options?.reviewWorkflows || false;
-
+function hasReviewWorkflow({ strapi }, contentType) {
+  if (typeof contentType === 'string') {
+    // If the input is a string, assume it's the modelUID of the content type and retrieve the corresponding object.
+    return hasReviewWorkflow({ strapi }, strapi.getModel(contentType));
+  }
+  // Otherwise, assume it's the content type object itself and return its `reviewWorkflows` option if it exists.
+  return contentType?.options?.reviewWorkflows || false;
+}
```

Comment:
wdyt of 
```suggestion
function hasReviewWorkflow({ strapi, contentType, uid }) {
  if (!isNil(uid)) {
    return hasReviewWorkflow({ strapi, contentType: strapi.getModel(contentType)});
  }
  
  if (isNil(contentType)) { throw Error... }
  
  // Otherwise, assume it's the content type object itself and return its `reviewWorkflows` option if it exists.
  return contentType?.options?.reviewWorkflows || false;
}
```
That way the method is more explicit on what the "contentType" is.

Reply:
I agree that the naming is a bit confusing inside the `hasReviewWorklflow` function, but it's localized and ease a lot the use of the function.
I don't want to ask people using the function to know which property they should fill in the parameter to get what they want. 🤔

Reply:
doesn't sound a bad argument to me!
Still most of people if they see contentType as an argument they will expect you have to pass the loaded model.

Reply:
Not a PR blocking comment but to me it makes more sense if the contentType argument is only of one type (e.g. uid === 'string')

I feel it makes the function tidier and easier to understand